test/L2ERC20.t.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Test} from "forge-std/Test.sol";
+import {L2ERC20} from "src/L2ERC20.sol";
+import {Constants} from "test/Constants.sol";
+
+contract L2ERC20Test is Test, Constants {
+  L2ERC20 erc20;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl("polygon_mumbai"));
+    erc20 = new L2ERC20("Hello", "WRLD", 0x0CBE91CF822c73C2315FB05100C2F714765d5c20);
+    erc20.registerApplicationContracts(
+      6, bytes32(uint256(uint160(0x76763243e202BB4153DDe5a7CE4013a9564f06Ad)))
+    );
+  }
+}
+
+contract ReceiveEncodedMsg is L2ERC20Test {
+  function testFork_CorrectlyDepositTokens() public {
+    erc20.receiveEncodedMsg(
```

Comment:
*RR* Similar as above—no test assertions. Also, there are things we could test here, like if invalid data is received it will revert, etc... Do we want test cases for those, or are these just meant as placeholders for now?

Reply:
Same as above